lib/Installation/Partitioner/LibstorageNG/TooSimplePasswordDialog.pm

```diff
@@ -0,0 +1,33 @@
+# SUSE's openQA tests
+#
+# Copyright © 2018 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved. This file is offered as-is,
+# without any warranty.
+
+# Summary: TBD
+# Maintainer: Oleksandr Orlov <oorlov@suse.de>
+
+package Installation::Partitioner::LibstorageNG::TooSimplePasswordDialog;
+use strict;
+use warnings FATAL => 'all';
+
+use PageInteractions;
+
+use parent 'Exporter';
+use Exporter;
+
+our @EXPORT = qw(
+    agree_with_too_simple_password
+);
+
+my $TOO_SIMPLE_PASSWORD_DIALOG = 'inst-userpasswdtoosimple';
```

Comment:
I guess lower case at least you aim to introduce some constant.

Reply:
Yes, wanted to introduce constant. 

The main idea was to keep needles in the top of a package as constants, so that it's easy to see how many needles are used in the particular module and to allow easily manage them in one place in case it they need to be changed.

Reply:
right, I think for `my` variables is lower letter and for constants we can group them like this:
```
use constant {
    FOO => "bar",
    BAR => "foo"
};
```